packages/test/test-end-to-end-tests/package.json

```diff
@@ -125,10 +125,10 @@
 		"uuid": "^8.3.1"
 	},
 	"devDependencies": {
+		"@fluid-internal/test-end-to-end-tests-previous": "npm:@fluid-internal/test-end-to-end-tests@2.0.0-internal.3.2.0",
```

Comment:
This will cause errors since this package doesn't exist. For this release you can try to use the existing published version, but I'm not sure how typetests work with unpublished (private) packages

Reply:
According to this PR, https://github.com/microsoft/FluidFramework/pull/14212/files#diff-d5f8decb3cbe6319402eaa606e81774d187e467cdbfaf7956823b65d23634d40 they stop generating typetests

Reply:
Got it. If that's the case, then we should probably remove the import/typetest section from package.json and remove the typetest directory as well?

Reply:
I dont see the typetest directory for mocha-test-setup